bin/laminas

```diff
@@ -33,6 +41,74 @@ $config = $container->get('config')['laminas-cli'] ?? [];
 
 $version = strstr(Versions::getVersion('laminas/laminas-cli'), '@', true);
 
+$eventDispatcher = new EventDispatcher();
+$eventDispatcher->addListener(
+    ConsoleEvents::TERMINATE,
+    static function (ConsoleTerminateEvent $event) use ($config) {
```

Comment:
Make this a class implementing `__invoke()`. It will make it easier to test, and make it easier to reduce nesting.

Reply:
Yes, that was the plan :)